### PR TITLE
cluster endpoint documentation

### PR DESCRIPTION
**Pull request by ankicabarisic**
This is documentation of the Kubernetes cluster endpoints, including initial cluster deployment, application management and cluster reconfiguration.

### CODE REVIEW

**Review by mbenguig — APPROVED** [review]
Nice work

File: endpoints/10-cluster-endpoints.md [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+🟡 POST {{protocol}}://{{sal_host}}:{{sal_port}}/sal/cluster/{{cluster_name}}/label
+```
+
+**Headers:** `sessionid`
+
+**Body:** Json input following this format:
+
+```json
+[
+  // to add lable it is to use value 'yes' and to remove 'no'
```
Comment (fviale) [inline endpoints/10-cluster-endpoints.md:292]: not sure the comment should be put inside the json format, as json does not support comments

I would rather write:
`
**Body:** Json input following this format ('yes' means a label will be added and 'no' means a label will be removed):

`

File: endpoints/10-cluster-endpoints.md [hunk 2]
```diff
@@ -0,0 +1,319 @@
...
+🟡 POST {{protocol}}://{{sal_host}}:{{sal_port}}/sal/cluster/{{cluster_name}}/scaleout
+```
+
+**Headers:** `sessionid`
+
+**Body:** Json input following this format:
+
+```json
+[
+  //scale out using WorkerNodeCandidate which is part of the cluster and introducing the new node name
```
Comment (fviale) [inline endpoints/10-cluster-endpoints.md:226]: json does not allow comments (similar remark as below).

You can do instead for example:
```

**Body:** Json input following this format:

Note: scale out using WorkerNodeCandidate which is part of the cluster and introducing the new node name

```json
[
...


```

File: endpoints/10-cluster-endpoints.md [hunk 3]
```diff
@@ -0,0 +1,319 @@
...
+    "packageManager" : "kubevela", // kubectl or helm
```
Comment (fviale) [inline endpoints/10-cluster-endpoints.md:163]: same comments as below, don't put comments inside json

File: endpoints/10-cluster-endpoints.md [hunk 4]
```diff
@@ -0,0 +1,319 @@
...
+      },
+      {
+        "nodeName": "{{worker_name}}",
+        "nodeCandidateId": "{{WorkerNodeCandidate}}",
+        "cloudId": "{{cloud_name}}"
+      }
+    ],
+    "env-var":{
+      //here add any env variable needed for the cluster in the form "ENV_VAR_NAME":"test-ENV_VAR_value"
+      // for instance: "APPLICATION_ID":"0fb75671-8955-4c06-9a8c-d397b29e3894"
```
Comment (fviale) [inline endpoints/10-cluster-endpoints.md:37]: same remarks as below

**Comment by ankicabarisic** [comment]
@Fabien Viale ***@***.***> I was adding this comments as
the all endpoint documentation is done in this way. Can update it now for
cluster but in that case should do same approach for all the endpoints (and
is not always trivial) ;)
Also same comments are valid in the postman.
Do you know if JSON could be maybe updated to support them, same as it does
in postman?

**Comment by fviale** [comment]
@ankicabarisic the json format does not allow comments, we can ask kindly the JSON standard to add them but you know ...

at any rate if it's the case for all endpoints then it's not the scope of this PR

**Review by fviale — APPROVED** [review]

**Pull request merged**